FunctionalTableData/Array+TableSection.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  Array+TableSection.swift
+//  FunctionalTableData
+//
+//  Created by Sherry Shao on 2018-06-14.
+//  Copyright © 2018 Shopify. All rights reserved.
+//
+
+extension Array where Element: TableSectionType {
```

Comment:
Can you put this file in the extensions folder?

Reply:
Fun fact, there never was a physical extension folder, just a project grouping called Extensions. I can create one.